modules/ixBidAdapter.js

```diff
@@ -103,6 +104,15 @@ function bidToImp(bid) {
 
   return imp;
 }
+/**
+ * 
+ * @param {*} number 
+ * @param {*} precision 
+ * @returns 
+ */
+function roundUp(number, precision) {
+  return Math.ceil((parseFloat(number) * Math.pow(10, precision)).toFixed(1)) / Math.pow(10, precision);
```

Comment:
can you round these down? rounding floors up seems potentially yield negative

roundUp(0.004,2)
0.01

Reply:
@patmmccann 
Should this also be changed in 
https://github.com/prebid/Prebid.js/blob/master/modules/priceFloors.js#L219 
?

Reply:
thats a good point @bszekely1 why did you choose to round floors up there?

Reply:
we talked this over in the design phase of floors, though it's been some time since we thought about it but the logic. The only place where rounding would likely cause any type of concern is when bids are adjusted for bidAdjustment or Currency Conversion. If anything, rounding up at 4 precision points helps the bidder. We would just round up the 4th precision decimal based on the 5th precision digit: https://github.com/prebid/Prebid.js/blob/b2cbab460f4392ea0b87fd59d78f0e07434ad300/modules/priceFloors.js#L58

At the precision of 4 decimal points, it's well below the CPM bucket that prebid normally assigns when bids come back in.

Reply:
also, 2 point precision rounding up is a not ideal since as you pointed out Patrick will cause very small numbers to equal a penny. At 4 points that is far enough away to compete fairly.